bitnami/redis/values.yaml

```diff
@@ -494,7 +494,14 @@ master:
     ## e.g.
     ## loadBalancerSourceRanges:
     ##   - 10.10.10.0/24
-    ##
+    ##    
+    externalIPs: []
```

Comment:
The description of each variable is set above , not below the variable itself. Also, the description of loadBalancerSourceRanges has been left on top of externalIPs but should be declared right above the variable itself. Could you fix it please?

Reply:
Yes, i did... now its like this:
```yaml
    ## @param master.service.loadBalancerIP Redis&reg; master service Load Balancer IP
    ## ref: https://kubernetes.io/docs/concepts/services-networking/service/#internal-load-balancer
    ##
    loadBalancerIP: ""    
    ## @param master.service.externalIPs Redis&reg; master service External IPs
    ## https://kubernetes.io/docs/concepts/services-networking/service/#external-ips
    ## e.g.
    ## externalIPs:
    ##   - 10.10.10.1
    ##   - 201.22.30.1
    ##
    externalIPs: []
    ## @param master.service.loadBalancerSourceRanges Redis&reg; master service Load Balancer sources
    ## https://kubernetes.io/docs/tasks/access-application-cluster/configure-cloud-provider-firewall/#restrict-access-for-loadbalancer-service
    ## e.g.
    ## loadBalancerSourceRanges:
    ##   - 10.10.10.0/24
    ##    
    loadBalancerSourceRanges: []
    ## @param master.service.annotations Additional custom annotations for Redis&reg; master service
    ##
```